src/anbox/cmds/session_manager.cpp

```diff
@@ -131,6 +131,11 @@ anbox::cmds::SessionManager::SessionManager()
                       use_system_dbus_));
 
   action([this](const cli::Command::Context &) {
+
+    Log().SetLogFile(std::string("/var/log/anbox/session.txt"));
```

Comment:
Using a static path here is wrong. We have a set of paths influenced by the cmake configuration already in config.h.in

Also this needs to take care of the correct path when running inside the snap. Please see src/anbox/system_configuration.cpp for details.

Thinking more about this, the path should come from a command line argument handed by the user. `--log-file=<path>` is what it should be.